src/nav-bar.js

```diff
@@ -2,36 +2,35 @@ import React from 'react';
 
 class NavBar extends React.Component {
     componentDidMount(){
-        /*
-        var fixedContainer = document.getElementById("fixed-container");
+        var fixedContainer = document.getElementById("navigation-container");
         var children = Array.from(fixedContainer.children);
 
         children.forEach(child => {
             var clone = child.cloneNode(true);
             clone.removeAttribute("id");
             clone.style.visibility = "hidden";
-
+            clone.classList.add("desktop-hide");
+            
             child.style.position = "fixed";
 
             fixedContainer.append(child);
             fixedContainer.append(clone);
         });
-        */
     }
     render(){
         return(
-            <div id="fixed-container">
-                <div className="desktop-banner mobile-hide" id="desktop-banner">
-                    <span>{this.props.title}</span>{this.props.content}
-                </div>
-                    
+            <div id="navigation-container">
                 <div className="navigation" id="navigation">
+                    <div className="desktop-banner mobile-hide" id="desktop-banner">
+                        <span>{this.props.title}</span>{this.props.content}
```

Comment:
Is the content not in a block element?

Reply:
There's some styling that points to this span & adjusts the font weight, do you think I should define it as its own class?
![image](https://user-images.githubusercontent.com/31863372/91087329-476db180-e61e-11ea-9b45-e16ad993a458.png)

Reply:
I mean, the `{this.props.content}` is just out in the HTML with no `p` tag or anything -- having text just sitting in HTML is not that pragmatic